pom.xml

```diff
@@ -73,14 +73,12 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <github.global.server>github</github.global.server>
     <maven-jar-plugin.version>2.4</maven-jar-plugin.version>
-    <jackson.groupId>org.codehaus.jackson</jackson.groupId>
-    <jackson.version>1.9.13</jackson.version>
-    <jackson.package>org.codehaus.jackson</jackson.package>
-    <shade.prefix>shaded.parquet</shade.prefix>
```

Comment:
Shading is used for other dependencies not only for jackson. Are you sure `shade.prefix` is not used elsewhere?

Reply:
It was still in there, but the ordering was changed. I've reverted it to its original position.

Reply:
Sorry, missed that. :)